WooCommerce/Classes/POS/TabBar/POSTabCoordinator.swift

```diff
@@ -174,14 +172,14 @@ private extension POSTabCoordinator {
             guard let self else { return }
 
             // Get local catalog eligibility as bool from service
-            // Service is created asynchronously in init, might not be ready yet
```

Comment:
When testing POS settings these past days, some of the times I didn't see the Local catalog card loading along the other settings. Do you think `await service.catalogEligibility(for: siteID)` might resolve this? I'll keep an eye in any case